src/ruby/lib/grpc/generic/active_call.rb

```diff
@@ -582,6 +584,23 @@ def attach_peer_cert(peer_cert)
 
     private
 
+    # response_status_as_trailers_only sends a status as Trailers-only
+    # https://github.com/grpc/grpc/blob/ceecf80283e6ca184df587f76ed053f1f5295b7f/doc/PROTOCOL-HTTP2.md
+    def response_status_as_trailers_only(code, details, metadata: {})
+      @send_initial_md_mutex.synchronize do
+        # Can't send Trailers-Only since Response-Headers have already been sent
+        return false if @metadata_sent
+        @metadata_sent = true
+      end
+
+      @call.run_batch(
+        SEND_INITIAL_METADATA => @metadata_to_send,
```

Comment:
Sorry, don't fully understand here - this `response_status_as_trailers_only` is sending both metadata and trailers, is this intended?

Reply:
Yes. I'd like to send response data as Trailers-Only (https://github.com/grpc/grpc/blob/ceecf80283e6ca184df587f76ed053f1f5295b7f/doc/PROTOCOL-HTTP2.md#responses) if possible.